### ISSUE TITLE
Could not load plugin

### ISSUE THREAD

**Issue by Ejrionm** [sebastien-bouffier/Bidoo#83]
Hi. I've downloaded the latest version from the release section (0.6.6), but when I load VCV (0.6.1), It says
"Could not load plugin Documents/Rack/plugins/Bidoo. See log for details"

What is happening?

Thanks

**Comment by davephillips** [comment]
Does the log refer to a missing symbol curl_easy_perform ? IIRC the binary links against the static libcurl provided by Rack's dep libs but when the plugin is loaded it uses the system libcurl (or it may be the other way around), which in my case does not include the missing symbol. Anyway, I added this line to the Makefile (between the FLAGS and SOURCES lines) to fix the issue and it works:

LDFLAGS += -L$(RACK_DIR)/dep/lib $(RACK_DIR)/dep/lib/libcurl.a

Try it, it works for me on Linux with the latest git pull from the Bidoo repo.

**Comment by sebastien-bouffier** [comment]
When you write 

> Could not load plugin Documents/Rack/plugins/Bidoo. See log for details

I suppose you are under Win. Do you confirm ? Which version ?

FYI I have a blank machine under W10 (blank means no dev lib/tools etc..) and 0.6.6 works fine with Rack 0.6.1 

Can you describe how you did the install ?

**Comment by sebastien-bouffier** [comment]
@Ejrionm I think that @davephillips raised the right issue. I am quite sure that curl (http lib) is the main reason of your issue. I do not have a W7 OS computer so I can not check why it doesn't work on your OS. Maybe when I build my pack against W10 it calls a socket lib which is different between W10 and W7 ... To understand why it doesn't work we need to run Rack in debug mode under your OS. Another information => when Rack 0.6.2 will be released my pack should be available thru Rack like the other plugins so ... Maybe it is worth to wait. If I can put my hands on a W7 computer I will try to find a solution.

**Comment by dirkleas** [comment]
macos: latest from plugin manager or build from source via sdk causes referenced error -- log:

```
[0.113 warn src/plugin.cpp:90] Failed to load library /Users/dirkleas/Documents/Rack/plugins/Bidoo/plugin.dylib: dlopen(/Users/dirkleas/Documents/Rack/plugins/Bidoo/plugin.dylib, 2): Symbol not found: _curl_easy_cleanup
  Referenced from: /Users/dirkleas/Documents/Rack/plugins/Bidoo/plugin.dylib
  Expected in: flat namespace
 in /Users/dirkleas/Documents/Rack/plugins/Bidoo/plugin.dylib
```

**Comment by lnikj** [comment]
Confirmed - latest plugin manager version broken for me too (OS X 10.11.6):

`[0.146 warn src/plugin.cpp:90] Failed to load library /Users/nik/Documents/Rack/plugins/Bidoo/plugin.dylib: dlopen(/Users/nik/Documents/Rack/plugins/Bidoo/plugin.dylib, 2): Symbol not found: _curl_easy_cleanup
  Referenced from: /Users/nik/Documents/Rack/plugins/Bidoo/plugin.dylib
  Expected in: flat namespace
 in /Users/nik/Documents/Rack/plugins/Bidoo/plugin.dylib`

**Comment by martin-lueders** [comment]
Same here on Mac OS X 10.13.6

**Comment by sebastien-bouffier** [comment]
Dear all, it looks like my pack has been released thru Rack before 0.6.2. #https://github.com/VCVRack/community/issues/418#event-1728967855
Curl dep needs to be built with `--disable-symbol-hiding` tag
I think it is the reason of you issue, I will get in touch with Andrew and Matt.

**Comment by ZuluEchoRomeoOscar** [comment]
Just a note to say I'm getting the same `undefined symbol: curl_easy_perform` with my Linux system.

**Comment by davephillips** [comment]
@ZuluEchoRomeoOscar Do you build Rack yourself and do you launch it with 'make run' ? If so you try the fix I recommended earlier in this thread, it's working fine for me on my old Fedora 23 system.

**Comment by sebastien-bouffier** [comment]
Dear all, Rack 0.6.2b has been released, do you still face the same issue when loading my pack ?

**Comment by lnikj** [comment]
No! :)

**Comment by sebastien-bouffier** [comment]
Great!!!

**Comment by dirkleas** [comment]
_curl_easy_cleanup reference issue resolved with Rack 0.6.2b -- great indeed!
